src/main/java/seedu/address/model/person/MeetingNameContainsKeywordsPredicate.java

```diff
@@ -0,0 +1,56 @@
+package seedu.address.model.person;
+
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.logging.Logger;
+
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.commons.util.StringUtil;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.model.meeting.Meeting;
+
+/**
+ * Tests that a {@code Person}'s {@code Name} matches any of the keywords given.
+ */
+public class MeetingNameContainsKeywordsPredicate implements Predicate<Person> {
+    private static final Logger logger = LogsCenter.getLogger(MeetingNameContainsKeywordsPredicate.class);
+    private final List<String> keywords;
+    public MeetingNameContainsKeywordsPredicate(List<String> keywords) {
+        this.keywords = keywords;
```

Comment:
Should we use an assertion here to test the function of the previous class that indeed keywords is not of length 0?

```suggestion
        assert(keywords.length > 0)
        this.keywords = keywords;
```